code/game/gamemodes/changeling/powers/fakedeath.dm

```diff
@@ -18,7 +18,7 @@
 	user.update_stat()
 	user.update_canmove()
 
-	addtimer(src, "ready_to_regenerate", LING_FAKEDEATH_TIME, TIMER_NORMAL, user)
+	addtimer(src, "ready_to_regenerate", LING_FAKEDEATH_TIME, TIMER_UNIQUE, user)

```

Comment:
This is unique per user! `addtimer` creates an internal `hashlist` which is used to determine whether or not the `timer` is unique.

`hashlist[1] = "[thingToCall](\ref[thingToCall])"`